src/main/java/com/google/sps/Trip.java

```diff
@@ -0,0 +1,190 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Date;
```

Comment:
Since Java 8 came out, the preference is to use the java.time libraries, not java.util for working with time. Oracle themselves recommend this (https://docs.oracle.com/javase/tutorial/datetime/iso/legacy.html).

Let me know if you have any questions about it!